src/axis.js

```diff
@@ -1,5 +1,15 @@
 import { timeFormat } from 'd3-time-format';
 
+export const getBreakpointLabel = (breakpoints, point) => {
```

Comment:
Move it to `index.js` as you use it here. But as rollup requires a single export in this file, we probably need an extra `breakpoint.js` file.